cargo-process.el

```diff
@@ -336,6 +349,24 @@ Without the prefix argument, return DEFAULT immediately."
       (read-shell-command "Cargo command: " default)
     default))
 
+(defun cargo-process--get-dependencies (manifest)
+  "Extract the list of dependencies from the
+MANIFEST (i.e. Cargo.toml)."
+  (save-excursion
+	(let ((buffer (find-file-noselect manifest))
+		  (deps-list))
+	  (set-buffer buffer)
+	  (setq buffer-read-only t)
+	  (widen)
+	  (goto-char (point-min))
+	  (search-forward "[dependencies]" nil t)
+	  (while (re-search-forward "^[a-z]* *=" nil t)
```

Comment:
This regexp doesn't cover capitals (not sure if packages have those?) dashes and underscores.